templates/2k11/admin/plugin_config.tpl

```diff
@@ -18,7 +18,7 @@
 
             <div id="el{$config_groupkeys@iteration}" class="config_optiongroup{if $config_groupkeys@last} config_optiongroup_last{/if} additional_info">
             {foreach $config_groupkeys AS $config_groupkey}
-                {if $plugin_option[$config_groupkey]['ctype'] == 'seperator'}
```

Comment:
Thats more like a bugfix? Maybe better single those out as separate commits

Reply:
Right. This whole part caused a lot of PHP 8 errors, which made me notice the code was wrong in the first place. Separate commits in the future!

Reply:
Separated into https://github.com/s9y/Serendipity/commit/ec70938faf6236dcc5f7b38c7c28373fd1200877